awsbill2graphite.py

```diff
@@ -457,11 +528,13 @@ def new_metric_ledger():
         TsRegionTotal(),
     ])
 
+
 def generate_metrics(csv_file, output_file):
-    """Generates metrics from the given CSV and writes them to the given file-like object."""
+    """Generates metrics from the given CSV and writes them to the given
+    file-like object."""
     reader = csv.reader(csv_file)
     col_names = reader.next()
-    formatter = MetricFormatter()
+    # formatter = MetricFormatter()
```

Comment:
I wasn't sure if you wanted to use this in the future. I just commented it out for now